File: tests/fixtures/golden/csharp/BareCallMethod.cs
namespace Golden.BareCallMethod
{
    public class Utility
    {
        // The only `Recalc` in this fixture -- a method-kind symbol.
        // Before #75, a receiver-less call to the same name would
        // fuzzy-bind to this via the bare-name fallback; #75 bars a bare
        // call (no receiver at all) from binding to any method-kind
        // candidate.
        public string Recalc() => "widget";
    }

    public class BareCaller
    {
        // Bare call, no receiver: Recalc is not declared in this class
        // and there is no `using static` bringing it into scope. Must
        // stay UNRESOLVED.
        public string CallBare() => Recalc();
    }
}

File: tests/fixtures/golden/typescript/bareCallMethod.ts
class Utility {
  // The only `recalc` in this fixture -- a method-kind symbol. Before
  // #75, a receiver-less call to the same name would fuzzy-bind to this
  // via the bare-name fallback; #75 bars a bare call from binding to any
  // method-kind candidate.
  recalc(): string {
    return "widget";
  }
}

// Bare call, no receiver: recalc is not declared or imported into this
// module. Must stay UNRESOLVED.
export function bareCaller(): string {
  return recalc();
}
